website/docs/guides/ttl-index.mdx

```diff
@@ -0,0 +1,40 @@
+---
+sidebar_position: 4
+description: Learn about TTL indexes in FerretDB.
+---
+
+# Time-To-Live (TTL) indexes
+
+import CodeBlock from '@theme/CodeBlock';
+import CreateIndex from '!!raw-loader!./requests/1-create-ttl-index.js';
+import InsertTTLData from '!!raw-loader!./requests/2-insert-ttl-data.js';
+
+TTL (Time-To-Live) indexes are useful when you need documents to expire automatically after a certain time. It's ideal for temporary records, logs, or session data.
+They are single-field indexes and not allowed as multi-field/compound indexes.
+
+:::note
+
+- Documents inserted before the index was created may not be affected until an insert or update operation occurs. It is recommended to create the TTL index before inserting documents.
+- The cleanup of expired documents is a background operation that runs every 60 seconds, so deletion may not be immediate.
+
+:::
+
+## Creating a TTL index
+
+A TTL index is created using the `createIndex` command with the field you want to index and the `expireAfterSeconds`.
```

Comment:
Do you mean "and the `expireAfterSeconds` option"?

Reply:
@Fashander I think it reads weird not knowing what `expireAfterSeconds` is

Reply:
It is documented two lines below, but I agree that "and the `expireAfterSeconds`." parts sound weird